src/call_frame.rs

```diff
@@ -29,13 +30,26 @@ pub struct CallFrame {
 pub struct Context {
     pub frame: CallFrame,
     pub near_call_frames: Vec<CallFrame>,
+    pub contract_address: Address,
```

Comment:
Please add a a doc comment for each new field here.

Reply:
Done, fyi, `context_u128` doc comes from [here](https://docs.zksync.io/zk-stack/components/zksync-evm/vm-specification#context-value).